src/backend/llvm_ast.c

```diff
@@ -432,6 +432,14 @@ struct LLVMOpaqueModule *blvm_create_module(
     if (compiler_args_print_backend_debug(ctx->args)) {
         bllvm_mod_debug(ctx->llvm_mod, mod_name);
     }
+
+    size_t mod_id_len;
+    char *mod_id = LLVMGetModuleIdentifier(ctx->llvm_mod, &mod_id_len);
```

Comment:
On my LLVM, `mod_name` always equals to `function_end` for some reason. Any ideas?